client/js/stores/envVarStore.js

```diff
@@ -0,0 +1,43 @@
+import lux from "lux.js";
+import { map, sortBy } from "lodash";
+
+function removeHalArtifacts( resp ) {
+	delete resp._links;
+	delete resp._origin;
+	delete resp._resource;
+	delete resp._action;
+	return resp;
```

Comment:
I worry that additional _ prefixed fields could be added to a hal response in the future. Could you use something like this here instead:
```js
function removeHalArtifacts( resp ) {
  return pick( resp, ( val, key ) { return key[ 0 ] !== "_" } );
}
```